main.c

```diff
@@ -1,6 +1,31 @@
+#include "mpi.h"
+#include "assistFunctions.h"
+#include "assistFunctions.c"
```

Comment:
shouldn't have to include the `.c` if you have the header file

Reply:
Done!